contracts/LiquidityBridgeContract.sol

```diff
@@ -113,7 +112,7 @@ contract LiquidityBridgeContract {
     bool private locked;
 
     mapping(bytes32 => uint8) private processedQuotes;
-    mapping(bytes32 => uint8) private processedPegOutQuotes;
+    mapping(bytes32 => uint8) public processedPegOutQuotes;
```

Comment:
I think this one should be private like the line above. By security reasons.

Reply:
when map is public, you can only see the values, you can't change.